plenum/common/messages/node_messages.py

```diff
@@ -191,6 +159,8 @@ class Prepare(MessageBase):
         (f.TXN_ROOT.nm, MerkleRootField(nullable=True)),
         (f.AUDIT_TXN_ROOT_HASH.nm, MerkleRootField(optional=True,
                                                    nullable=True)),
+        (f.ORIGINAL_VIEW_NO.nm, NonNegativeNumberField(optional=True,
+                                                       nullable=True)),
```

Comment:
If we are going to stop resetting pp_seq_no after view change then I suspect this new parameter might be not necessary.

Reply:
Well, we need old view only to be able to apply batches in audit ledger correctly. 
So, yes, most probably it's not needed in Prepares and Commits, but we need to think if it's needed, for example, in NewView's batchIDs.

Reply:
Yes, I agree that this needs to be checked, preferably using tests. This is clearly out of scope of this PR, just dropping a reminder.